gtsam/geometry/Pose3.cpp

```diff
@@ -183,16 +183,35 @@ Pose3 Pose3::Expmap(const Vector6& xi, OptionalJacobian<6, 6> Hxi) {
   // Get angular velocity omega and translational velocity v from twist xi
   const Vector3 w = xi.head<3>(), v = xi.tail<3>();
 
+  // Instantiate functor for Dexp-related operations:
+  const bool nearZero = (w.dot(w) <= 1e-5);
```

Comment:
`bool nearZero = (w.dot(w) <= nearZeroThreshold);` down below

Reply:
Is this intended?

Reply:
Yes. This has not changed.

Reply:
This PR is threshold neutral. It's not configurable nor consistent, but this PR does not change that in any way and doe not intend to fix it.